test/integration/virtual-rules/index.js

```diff
@@ -0,0 +1,315 @@
+describe('virtual rules', function() {
+	describe('checks', function() {
+		describe('autocomplete-valid', function() {
+			it('should pass', function() {
+				var results = axe.runVirtualRule(
+					'autocomplete-valid',
+					new axe.SerialVirtualNode({
```

Comment:
We don't need to create a virtual node, this is done by runVirtualRule.

```suggestion
					{
```